packages/server/src/queue/queue.entity.ts

```diff
@@ -71,12 +71,7 @@ export class QueueModel extends BaseEntity {
   isOpen: boolean;
 
```

Comment:
(this comment isnt related to this line, I'm just putting it here so we can thread responses)
It looks like queues with no staff members now no longer show up for professors on their "check in" modal (the one where they can pick which queue to check into). Can you look into why that's happening 🤔 

Also not sure if this would interfere with @VKong6019 and @tiingweii-shii with their work on extending ad hoc queues to TAs, although you guys are only on the TA side of things right